presto-accumulo/src/test/java/io/prestosql/plugin/accumulo/TestAccumuloIntegrationSmokeTest.java

```diff
@@ -33,28 +34,22 @@ protected QueryRunner createQueryRunner()
         return createAccumuloQueryRunner(ImmutableMap.of());
     }
 
+    @Test
```

Comment:
technically, it's redundant

Reply:
Agreed, but it seems cleaner to have them. For example, what if the original had something like `@Test(expectedExceptions=...)`